specification/dnsresolver/resource-manager/Microsoft.Network/stable/2025-05-01/dnsresolverpolicy.json

```diff
@@ -1593,6 +1654,11 @@
             "type": "string"
           }
         },
```

Comment:
you might want to mention whether its possible to have both of these properties or if they're exclusive?